dashboard/test/mailers/pd/workshop_mailer_test.rb

```diff
@@ -66,4 +66,14 @@ class WorkshopMailerTest < ActionMailer::TestCase
       Pd::WorkshopMailer.exit_survey(enrollment).deliver_now
     end
   end
+
+  test 'organizer should close reminder email has correct url' do
+    workshop = create :pd_workshop, num_sessions: 1
+    email = Pd::WorkshopMailer.organizer_should_close_reminder(workshop)
+    html = Nokogiri::HTML(email.body.to_s)
+    links = html.css('a')
+
+    assert_equal 1, links.length
+    assert links[0]['href'].include?('studio.code.org')
```

Comment:
Since I will be writing more of these unit tests, 'studio.code.org' and 'code.org' should be constants somewhere. Where would make the most sense to put them?

Reply:
You should just be able to use `CDO.code_org_url` and `CDO.studio_url`. In fact, you'll want to do that rather than hardcoding the path so it works in other environments.

Reply:
Rather, like this: `CDO.code_org_url('', CDO.default_scheme)`